### DISCUSSION TITLE
Setting the format path

### DISCUSSION THREAD

**Discussion by darionyaphet** [juicedata/juicefs#254] [Q&A] [answered]
**What would you like to be added**:

when we format the file system I find it's use the `/var` as default path.  Can I add a parameter to control it?

**Why is this needed**:

**Comment by davies** [comment, marked as answer]
When storage engine is `file`, you can use `--bucket` to specify the root dir

**Reply by xiaogaozi** [reply]
@darionyaphet Is your problem solved?

**Reply by darionyaphet** [reply]
thanks a lot this problem seems to be solved with the following command :

```
./juicefs format -bucket /tmp localhost test
```
